packages/boxel-ui/addon/components/header/index.gts

```diff
@@ -20,11 +20,11 @@ interface Signature {
 
 const Header: TemplateOnlyComponent<Signature> = <template>
   <header
-    class={{cn
-      'boxel-header'
-      boxel-header--no-background=@noBackground
-      boxel-header--highlighted=@isHighlighted
-      boxel-header--large=(or (bool @title) (eq @size 'large'))
+    {{!-- Can the first argument be optional? --}}
```

Comment:
It looks like you could get that effect by skipping `cn` and directly calling the library that it calls.